packages/code-du-travail-frontend/src/common/Article.js

```diff
@@ -5,33 +5,51 @@ import styled from "styled-components";
 import {
   Container,
   PageTitle,
-  Subtitle,
+  Subtitle as Suptitle,
```

Comment:
this is so weird :D

Reply:
Je suis d'accord mais à part un gros refactor sur la manière donc je nomme les en-têtes de titre (sur plusieurs components dans l'UI) c'est la seule douille de naming que j'ai réussi à trouver pour les faire cohabiter

Reply:
Pas de soucis, ca m'a surtout fait rire